### DISCUSSION TITLE
How to update

### DISCUSSION THREAD

**Discussion by mjl38** [OpenSourcePFCLibraries/2021#7] [Q&A] [answered]
Hi, 
I've watched the video as indicated in the README and followed the instructions. What's unclear to me, is whether I'm missing something. The ws_objects folder has been updated, but the individual folders with the PBL files and the downloadable ZIP file haven't. Am I missing some step?
regards
MiguelL

**Comment by bruce-armstrong** [comment, marked as answer]
There's no continuous integration configured.  In order to get the PBLs and ZIP files updated somebody has to do a pull of the latest code and then create a release.

**Reply by mjl38** [reply]
So do I have to create a new fork or can I use the one that I already have in my repository?

**Reply by bruce-armstrong** [reply]
You can use your fork as long as it's current with the main branch.

**Reply by mjl38** [reply]
Okay, I'll try to do that in a few hours time (with some coffee, it's 2 AM here).
Thanks!

**Reply by mjl38** [reply]
All done. I think and hope it's correct. I've added the ZIP when creating the tag, but didn't add it again when creating the pull request. 
regards

**Reply by bruce-armstrong** [reply]
You've updated the PBLs.  If you want to create a new release though try this page and see if "Draft a new release" is available to you:

https://github.com/OpenSourcePFCLibraries/2021/releases

**Reply by mjl38** [reply]
Nope, not available, but that might have something to do with me deleting my fork? (as suggested by Github). I'll probably have to start over?

**Reply by bruce-armstrong** [reply]
No, you just didn't have sufficient privs.  I just sent you and invite that should give you the privs.

**Reply by mjl38** [reply]
Thanks for the invite. I've created the draft and added a single ZIP file with everything in it. Not sure if that's correct. Then ... how do I now 'publish' this release?

**Reply by mjl38** [reply]
I think I figured it out. It didn't like my Tag name. It seems to have been published now.
Many thanks!

**Reply by bruce-armstrong** [reply]
Good work!
